common/test/data/unicode_graded/README.md

```diff
@@ -0,0 +1 @@
+This is a very simple course for testing student responses reports for courses with unicode information.
```

Comment:
Can you change this to describe what the course contents are, rather than how you're using it? What is the "unicode information"? Is it HTML? Student answers? What types of problems are there? etc.